deadcode.go

```diff
@@ -12,13 +12,25 @@ import (
 )
 
 var exitCode int
+var ignoreNames []string
+
+func init() {
+	flag.String("dir", ".", "directories to search, example: Foo1 Bar2")
+	flag.String("ignore", "", "sourcefile:name to ignore, example: somefile.go:aVariableFoo anotherfile.go:aConstBar")
+}
 
 func main() {
 	flag.Parse()
-	if flag.NArg() == 0 {
+
+	if flag.Lookup("ignore") != nil {
+		for _, name := range strings.Fields(flag.Lookup("ignore").Value.String()) {
+			ignoreNames = append(ignoreNames, name)
+		}
+	}
+	if flag.Lookup("dir").Value.String() == "." {
```

Comment:
Is it possible to have `dir` be the default operation, without requiring a `-dir` argument? It seems logical that it shouldn't need one, since it is the default argument, whereas `-ignore` is more of an exception.